my-app/src/Components/About/About.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Mypic from '../../images/mypic.jpeg';
```

Comment:
`Mypic`

usual variables should be made in `lowe case`. Only classes and components start with a capital letter